politeiawww/paywall.go

```diff
@@ -212,10 +215,21 @@ func (b *backend) checkForProposalPayments(pool map[uuid.UUID]paywallPoolMember)
 			continue
 		}
 
-		// Removed paywall from the in-memory pool if it has been marked as paid.
+		// Removed paywall from the in-memory pool if it has
+		// been marked as paid.
 		if !b.userHasValidProposalPaywall(user) {
 			userIDsToRemove = append(userIDsToRemove, userID)
 			log.Tracef("  removing from polling, user just paid")
+		} else if tx != nil {
+			// Update pool member if payment tx was found but
+			// does not have enough confimrations.
+			poolMember.txID = tx.ID
+			poolMember.txAmount = tx.Amount
+			poolMember.txConfirmations = tx.Confirmations
+
+			b.Lock()
+			pool[userID] = poolMember
```

Comment:
I think it should be
`b.userPaywallPool[user.ID] = poolMember`
Since the pool is a scoped variable and won't affect the real map, right?

Reply:
Fixed.